README.md

```diff
@@ -90,7 +90,7 @@ The syntax of a result is simple.
 `{ title | action }`or `{ title | action | description }`
 The `title` is displayed in the results and the `action` is written to standard out
 when that result is selected.  A common use case would therefore be
-`lighthouse | sh` and `action` would be some shell command.  Run `make config` and then
+`lighthouse | sh` and `action` would be some shell command.  Run `lighthouse-install` and then
```

Comment:
some of the readme will need cleaning up if `make install` is already run (i.e. removing instances of `make config` rather than replacing it with `lighthouse-install`).  when would a user ever need to run lighthouse-install if they follow the guidelines?